app/presenters/bill-licences/view-standard-charge-transaction.presenter.js

```diff
@@ -174,6 +176,7 @@ function _presrocContent (transaction) {
 }
 
 function _srocContent (transaction) {
+  console.log('🚀 ~ _srocContent ~ transaction:', transaction)
```

Comment:
Think you forgot to remove your `console.log`

Reply:
Doh! Good spot!